### DISCUSSION TITLE
friendly urls

### DISCUSSION THREAD

**Discussion by Friendly-Hackerr** [community/community#53446] [Programming Help]
### Select Topic Area

Question

### Body

Hello, I have a question about how to enter friendly URL links on the website, i.e. instead of "index.html" it was "Home-page"

**Comment by emullallphotography** [comment]
To enter friendly URL links on a website, you can follow these steps:

1. Ensure that your website is built using a content management system (CMS) or a web development framework that supports friendly URLs.
2. Identify the page or content you want to create a friendly URL for, such as the homepage.
3. Determine the desired friendly URL structure, such as "home-page" instead of "index.html".
4. Access the settings or configuration options of your CMS or web development framework.
5. Look for the option to customize or set the URL for the specific page or content.
6. Enter the desired friendly URL, such as "home-page", and save the changes.
7. Test the new friendly URL by accessing it in a web browser to ensure it properly redirects to the intended page.
